src/pages/Calendar/index.tsx

```diff
@@ -1,19 +1,61 @@
 import useHours from '@/hooks/useHours';
+import useMe from '@/hooks/useMe';
+import usePdf from '@/hooks/usePdf';
+import useUserHours from '@/hooks/useUserHours';
 import { Hours } from '@/types/Hours';
 import dayGridPlugin from '@fullcalendar/daygrid';
 import interactionPlugin from '@fullcalendar/interaction';
 import FullCalendar from '@fullcalendar/react';
 import timeGridPlugin from '@fullcalendar/timegrid';
-import { useState } from 'react';
+import { useRef, useState, useEffect, useCallback } from 'react';
 
+import PdfFetcher from '../Users/components/PdfFetcher';
+import SelectUsers from '../Users/components/selectUsers';
 import HoursForm from './HoursForm';
 
 export function Calendar() {
   const [selectedDate, setSelectedDate] = useState<string | null>(null);
   const [isDialogOpen, setDialogOpened] = useState(false);
   const [event, setEvent] = useState<Hours | null>(null);
+  const [selectedUser, setSelectedUser] = useState<string | null>(null);
+  const [currentMonth, setCurrentMonth] = useState<string>('2024-06');
+  const { me } = useMe();
+  const { userHours } = useUserHours(
+    selectedUser || '074221c8-1545-4e4b-a241-60addeaa0764'
```

Comment:
co to za UUID, nie powinno być czegoś takiego, parametr opcjonalny

Reply:
\+